.github/workflows/build_wheels_macos.yml

```diff
@@ -187,7 +187,8 @@ jobs:
           source "${BUILD_ENV_FILE}"
           ${CONDA_RUN} pip install awscli
           for pkg in dist/*; do
-            ${CONDA_RUN} aws s3 cp "$pkg" "s3://pytorch/whl/${CHANNEL}/${{ matrix.desired_cuda }}/" --acl public-read
+            # PYTORCH_S3_BUCKET_PATH derived from pkg-helpers
```

Comment:
I think we'll need to pass `UPLOAD_TO_BASE_BUCKET` as an env var to this workflow too right?

Reply:
Probably not, the default is no and we don't have cuda builds for macOS so I think it should be fine